lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -561,6 +561,60 @@ ospf:
     config_get_token: ['/^router ospf <name>$/i', '/^vrf\s+(\S+)$/']
     config_set: ["router ospf <name>", "<state> vrf <vrf>", "end"]
 
+radius_server:
+  hosts:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/'
+    config_set: '<state> radius-server host <ip>'
+    default_value: ''
+
+  auth-port:
+    config_get: "show running-config radius"
```

Comment:
@jonnytpuppet Yes we did agree to that which can still be accomplished using the 'show running-config radius all'.  If you recall from my last email, the inherited values like 'timeout' and 'retransmit' are NOT shown on individual hosts.  Only the system defaults for 'auth-port' and 'acct-port' are shown but they are not configurable at the global level and therefore not inheritable.  If you don't use the 'SRC radius all' you won't be able to pick up the per/host default values for 'auth-port' and 'acct-port'.  

For example:

1) Starting default values
```
dt-n9k5-1(config)# do sh run radius all

!Command: show running-config radius all
!Time: Tue Oct 20 14:29:15 2015

version 7.0(3)I2(1)
radius-server test username test password test idle-time 0 
radius-server timeout 5
radius-server retransmit 1
radius-server deadtime 0
aaa group server radius radius 
    deadtime 0
    use-vrf default
    no source-interface

no ip radius source-interface
dt-n9k5-1(config)# 
```
2) Configure host, but don't specify any override values

```
dt-n9k5-1(config)# radius-server host 5.5.5.5
warning: no key is configured for the host
dt-n9k5-1(config)# 

dt-n9k5-1(config)# do sh run radius all | inc 'radius-server host 5.5.5.5'
radius-server host 5.5.5.5 auth-port 1812 acct-port 1813 authentication accounting 
radius-server host 5.5.5.5 test username test password test idle-time 0
```

As you can see above,  `auth-port` and `acct-port` are displayed but if you only use the 'show running radius' command those values do not get displayed.

```
dt-n9k5-1(config)# do sh run radius | inc 'radius-server host 5.5.5.5'
radius-server host 5.5.5.5 authentication accounting 
dt-n9k5-1(config)# 
```